### PR TITLE
Added CodeForces Scraper

### LINKED ISSUE #34: Code Forces problem scraper
# Description

Scrape the problem statement from the Code Forces website as per the problem code provided by the user.
Save the result in a text file.

## Type of issue
- [x] Feature (New Script)

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by roopeshvs**
# Description

Added a Script to scrape CodeForces Problem Statement with Input & Output Examples of the question, given a CodeForces Problem Code via the Command-Line.

Fixes #34 

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [ ] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Web_Scrappers/CodeForces_Scraper/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Run The Command 
+
+`pip install -r requirements.txt`
+
+## Instructions To Run
+
+Run The Command
+
+`python codeforces_scraper.py`
```
Comment (AdityaJ7) [inline Scripts/Web_Scrappers/CodeForces_Scraper/README.md:15]: Here the problem code must also be given

File: Scripts/Web_Scrappers/CodeForces_Scraper/codeforces_scraper.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        if not page:
+            raise Exception(page.status_code)
+    except Exception as e:
+        print("Cannot Find CodeForces Problem!" + str(e))
+        exit(0)
+    soup = BeautifulSoup(page.content, 'html.parser')
+    soup.find('div', class_ = 'header').decompose()
+    problem_statement_div = soup.find('div', class_ = "problem-statement")
+    response = problem_statement_div.text.replace("$$$", "")
+    row = response.split(' ')
```
Comment (tusharnankani) [inline Scripts/Web_Scrappers/CodeForces_Scraper/codeforces_scraper.py:30]: This variable is never used.

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Please make the requested changes in the Readme and add a screenshot of the script in action

The script works fine, Good Job @roopeshvs

File: Scripts/Web_Scrappers/CodeForces_Scraper/README.md (hunk 1 above)
Comment (tusharnankani) [reply Scripts/Web_Scrappers/CodeForces_Scraper/README.md:15]: Yes!

**Review by tusharnankani — CHANGES_REQUESTED** [review]
Also, please add a sample.txt file, in addition to the working screenshot of the script. 
Thanks!

**Review by AdityaJ7 — APPROVED** [review]
Good Job @roopeshvs

**Review by tusharnankani — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @roopeshvs  for code

**Pull request merged**